homework-solutions/js/hw-4/task-2.js

```diff
@@ -14,3 +14,25 @@
   5. Проверить задание со следующими значениями в переменной age: 10, 17, 18, 19, 59, 60, 61
 
   */
+
+let minAge = 18;
+let maxAge = 60;
+let age = 'k';
+
+const incorrectTypeMessage = 'Incorrect data type';
+const defaultMessage = 'Technical work';
+const number = 'number';
+
+if (typeof age !== number) {
```

Comment:
```suggestion
if (typeof age !== "number") {
```
Иначе number воспринимается как переменная

И лишние переменные тут не нужны

Reply:
fixed